agoradatatools/etl/utils.py

```diff
@@ -1,19 +1,34 @@
 import synapseclient
 import yaml
-import errno
-import sys
 
 
-def _login_to_synapse(authtoken: str = None) -> object:
+def _login_to_synapse(token: str = None) -> synapseclient.Synapse:
+    """Logs into Synapse python client, returns authenticated Synapse session.
+
+    Args:
+        authtoken (str, optional): Synapse authentication token. Defaults to None.
+
+    Returns:
+        synapseclient.Synapse: authenticated Synapse client session
+    """
     syn = synapseclient.Synapse()
-    if authtoken is None:
+    if token is None:
         syn.login()
     else:
-        syn.login(authToken=authtoken)
+        syn.login(authToken=token)
     return syn
 
 
-def _get_config(config_path: str = None):
+def _get_config(config_path: str = None) -> list:
+    """Takes config_path and opens yaml file path points to, loads configuration from file.
+    If no config_path is supplied, defaults to "./config.yaml"
+
+    Args:
+        config_path (str, optional): Path to config file. Defaults to None.
+
+    Returns:
+        list: list of dictionaries containing configuration information for run.
+    """
     if not config_path:
```

Comment:
Nit: Change this to `config_path is None:`

Reply:
both addressed in latest commit